Ginger/GingerCore/ALM/QtestCore.cs

```diff
@@ -74,7 +74,18 @@ public override bool ConnectALMServer()
         public override bool ConnectALMProject()
         {
             ALMCore.DefaultAlmConfig.ALMProjectName = ALMCore.DefaultAlmConfig.ALMProjectKey;
-            return true;
+            if (string.IsNullOrEmpty(ALMCore.DefaultAlmConfig.ALMServerURL) == false &&
+                string.IsNullOrEmpty(ALMCore.DefaultAlmConfig.ALMUserName) == false &&
+                string.IsNullOrEmpty(ALMCore.DefaultAlmConfig.ALMPassword) == false &&
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6008688)